deploy/install.sh

```diff
@@ -207,14 +257,24 @@ oc apply -f $KABANERO_SUBSCRIPTIONS_YAML --selector kabanero.io/install=13-subsc
 checksub openshift-pipelines openshift-operators
 checksub appsody-operator-certified openshift-operators
 
-# Install 14-subscription (che, kabanero)
+# Install 14-subscription (codeready-workspaces, kabanero)
+
+# Prior to installing codeready-workspaces, remove all eclipse-che instances and 
+# eclipse-che operator deployments in the kabanero namespace.
+removeResourceInstance checlusters.org.eclipse.che CheCluster kabanero
+unsubscribe eclipse-che kabanero
```

Comment:
I am just wondering, if we're going to have to go thru and clean up all of the stale CSVs..... hmm no we're not because Che was just installed at the namespace scope.  I guess we should just keep this in mind for the future, if we ever remove a cluster scope operator in the install script, there is a boat-load of extra work to be done to clean up the mess.